src/plugins/raven/services/Raven.js

```diff
@@ -217,22 +217,32 @@ export default class Raven {
   onError(...args) {
     const state = this.store.getState();
     const accessToken = getAccessToken(state);
-    const error = args.find((arg) => arg instanceof Error) || args[0];
-    const logData = {
-      app_version: APP_VERSION,
-      error: JSON.stringify(error, Object.getOwnPropertyNames(error)).replaceAll('://', '[PROTOCOL_TOKEN]').replace(/(\/\.\.)+\/?/g, '[UP_DIRECTORY_TOKEN]'),
-      request_uri: window.location.href.replace('://', '[PROTOCOL_TOKEN]'),
-    };
-    const errorHash = md5(logData.error);
+    const errorData = args.find((arg) => arg instanceof Error) || args[0];
 
+    let error;
+    try {
+      error = JSON.stringify(errorData, Object.getOwnPropertyNames(errorData || {}));
+    } catch (e) {
+      console.error('raven::onError::error', e);
+    }
+
+    error = (error || '').replaceAll('://', '[PROTOCOL_TOKEN]').replace(/(\/\.\.)+\/?/g, '[UP_DIRECTORY_TOKEN]');
```

Comment:
@hunyoboy dont think it is super helpful to log an empty error to the stream - what about a message like "failed stringifying errorData" or something?

Reply:
good morning. it won't due to L235. tested by throwing an error anywhere in blocksmith to trip `stringify`  :
 
 `const err = new Error(‘test’);
  err.me = err;
  throw err;`

or make ` stringify` return undefined.